app/_locales/en/messages.json

```diff
@@ -1576,6 +1576,9 @@
   "mainnet": {
     "message": "Ethereum Mainnet"
   },
+  "mainnetToken": {
+    "message": "This is a mainnet token"
```

Comment:
Perhaps to resolve @darkwing's point about design, we could just modify this text a bit. Change it to "This is a mainnet token address. Adding it to the current network is likely to fail."

Reply:
Done.